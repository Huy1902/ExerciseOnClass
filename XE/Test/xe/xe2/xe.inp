150 200
13067 17081 6431 16097 11604 9977 7265 31865 29487 11332 1132 10756 20291 19608 30033 20079 10946 2584 3413 9902 25509 28282 17652 785 6232 23464 6113 26928 3705 26986 1649 23460 7034 31603 8951 30999 18202 28425 15733 30962 3053 31675 19420 21693 17203 23551 23955 26263 9090 31141 23942 29748 28341 23950 7486 22697 26651 21295 15765 868 18889 13278 12197 8958 2113 13506 17701 29109 28030 20988 15900 10380 12398 8450 18275 16481 11035 8675 25072 15918 22594 6772 29952 11470 22353 18362 642 4581 30958 22514 14461 21929 5918 1416 7764 20703 5833 15568 14352 24669 7992 8089 8191 27564 30438 10968 22906 31912 30009 12971 25762 3496 24121 8177 15215 17176 22640 4931 22323 26575 4359 28661 5355 31391 9311 19513 21385 12066 11236 30135 21196 21262 8116 23526 10865 6389 31062 20820 13260 4241 22215 7158 15898 18720 24600 16516 22440 23310 10600 7369 
11333 7144 7408 6448 31724 27134 22324 28518 14775 10548 11204 28142 12421 16702 9672 21389 22902 21739 1892 20016 9501 3437 31085 7045 29468 616 1257 29356 21970 855 10931 3815 24755 30390 24630 24685 25333 5909 5575 9816 16422 2835 19823 11351 20726 12473 9479 22568 15162 26722 9835 1628 1341 14329 14238 4534 9125 5754 3455 24807 1681 21265 1836 29192 16913 31976 29214 14800 30867 26394 14728 751 11554 9506 9933 8536 23221 20920 15983 15425 22980 18532 8957 8796 7798 19442 24580 2262 6926 104 1181 21170 25544 20663 11309 11639 12859 13337 27351 15924 25627 15859 12100 31600 1816 21896 15520 6315 22698 17057 27859 30484 14385 20589 15578 31621 26389 11036 942 20249 20060 31317 9452 23712 30182 11398 27924 14140 21794 12922 31630 9976 21989 30612 31515 11920 7204 22706 10042 15039 3133 24895 2007 21585 3609 27771 22934 17692 18445 6464 21554 25272 21711 31598 18012 17251 30635 15661 10414 4773 28538 27681 6548 22979 25420 4868 1949 25709 18062 11258 4204 67 8815 14096 25804 12452 19514 30662 30275 5844 22891 2653 14023 24780 15002 13045 16960 743 4403 2697 5240 16717 14982 8965 26003 28034 23133 8535 30821 11942 