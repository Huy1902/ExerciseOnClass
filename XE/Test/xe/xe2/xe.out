21194933100
71 136 92 13 140 192 114 90 143 80 81 79 132 159 105 139 102 181 109 65 166 141 126 77 16 89 151 32 7 145 147 24 45 188 148 91 1 190 186 27 138 26 31 123 73 150 38 56 110 30 4 167 42 108 121 3 171 21 130 197 118 54 193 156 17 55 44 88 22 40 46 14 68 169 95 103 115 155 191 176 198 78 19 49 173 170 124 106 53 76 174 57 144 164 20 75 153 98 128 39 43 177 149 59 52 158 137 172 63 9 160 48 180 82 183 96 85 129 83 189 29 182 18 72 187 10 47 185 107 61 15 74 86 163 100 62 119 51 113 133 194 94 97 11 58 200 84 2 41 120 